app/views/documents/show/_requirements.html.erb

```diff
@@ -1,13 +1,15 @@
 <% issue_params = {
   style: "summary",
-  link_options: {
-    title: { href: "#content" },
-    summary: { href: "#content" },
-    body: { href: "#content" },
-    change_note: { href: "#content" },
-    alt_text: { href: "#lead-image" },
-    caption: { href: "#lead-image" },
-    topics: { href: "#topics" }
+  hrefs: {
```

Comment:
It looks like this have overwritten the recent change to switch to link options. See https://github.com/alphagov/content-publisher/pull/1076/commits/32027cfde1e495f9fee337b1a720ea618c8963ab

Aside from the overwrite they look good though 👍